### DISCUSSION TITLE
Responsive offcanvas and need of `data-bs-target="#offcanvasResponsive"` for close button

### DISCUSSION THREAD

**Discussion by julien-deramond** [twbs/bootstrap#36626] [Q&A]
I'm creating this discussion just in order to not to forget this question.

Why is `data-bs-target="#offcanvasResponsive"` needed only in the [Responsive Offcanvas example](http://localhost:9001/docs/5.2/components/offcanvas/#responsive) for the close button?
Does it hide a bug or is it normal?

https://github.com/twbs/bootstrap/blob/eea1b1e1fd04596265cad9d8036a73d20c7a3c5d/site/content/docs/5.2/components/offcanvas.md?plain=1#L170

Without it, an error can be seen in the console:
```
14:53:26.365 Uncaught TypeError: this._config is undefined
    _initializeBackDrop offcanvas.js:179
    Offcanvas offcanvas.js:71
    getOrCreateInstance base-component.js:65
    enableDismissTrigger component-functions.js:25
    handler event-handler.js:118
    addHandler event-handler.js:184
    on event-handler.js:217
    enableDismissTrigger component-functions.js:15
    <anonymous> offcanvas.js:275
    <anonymous> bootstrap.bundle.js:9
    <anonymous> bootstrap.bundle.js:10
2 offcanvas.js:179:22
```

**Comment by joel-porquet** [comment]
Thank you so much! I just spent a couple of hours not understanding why my close button was ineffective, and your message made me see what was missing. I wish the documentation insisted on that particular aspect, instead of just showing it in the related example.
